Makefile

```diff
@@ -74,10 +74,10 @@ IMAGEPATH := $(PKGDATADIR)/clear-containers.img
 KERNELPARAMS :=
 
 # The CentOS/RHEL hypervisor binary is not called qemu-lite
```

Comment:
Just one more change, please remove CentOS from the comment and rename `qemu-lite` to `qemu-cc`